index.js

```diff
@@ -20,6 +22,109 @@ app.get('/info', (req, res) => {
   res.send({ serverName: packageData.name, serverVersion: packageData.version });
 });
 
+// route for displaying all products
+app.get('/products/all', (req, res) => {
+
+  var i, j;
```

Comment:
Don't use this as scope variables. Initialize inside the loops if necessary.

Reply:
Thanks for pointing out that, Done!